src/App.css

```diff
@@ -106,13 +128,55 @@ a:visited {
   list-style-type: none;
   padding: 10px;
   margin: 10px;
-  background-color: #ff165d;
+  background-color: var(--pink);
   border-radius: 8%;
   text-decoration: line-through;
   cursor: pointer;
   opacity: 80%;
 }
 
+.listFrame {
+  display: flex;
+  flex-wrap: wrap;
+  box-shadow: 2px 2px 10px var(--black);
+  border-radius: 2px;
+  width: 90vw;
+  min-height: 90vh;
+  height: 100%;
+  justify-content: space-evenly;
+  background-color: var(--green);
+  border: 1px solid var(--black);
+  padding: 5vw;
+}
+
+.listFrame > h1 {
+  flex: 0 0 100%;
+}
+
+#soonItems li {
+  background-color: var(--white);
+}
+
+#kindOfSoonItems li {
+  background-color: var(--brown);
+}
+
+#notSoonItems li {
+  background-color: var(--red);
+}
+
+#inactiveItems li {
+  background-color: var(--yellow);
+}
```

Comment:
I see some room for improvement here!

- You should avoid using IDs in CSS, that's generally a 👃 - IDs has so much weight that it's difficult to overwirte an style if you need to. For this I'd prefer using classes instead.
- Targeting HTML elements is not also a good practice, since right now, you're saying `Every h1 inside .listFrame should have flex: 0 0 100%;` but what would happen if you decide to stop using `h1` and instead, start using an `h2`? To fix this, I'd suggest replacing the `h1` with a class, maybe something like `list-title` and then do

```css
.listFrame > .list-title { .... }
```